schema/SCHEMA_QUDT-v2.1.ttl

```diff
@@ -1047,24 +1047,12 @@ qudt:Quantity
       owl:allValuesFrom qudt:QuantityValue ;
       owl:onProperty qudt:quantityValue ;
     ] ;
-  rdfs:subClassOf [
```

Comment:
We still want Quantity to point to QuantityKind.
We don't want QuantityValue to point to QuantityKind.

Reply:
This is done

Reply:
I'm not seeing Quantity pointing to QuantityKind.